MacOS `cargo test` build fails

`cargo tests` fails with:

```
   Compiling zallet v0.0.0 (/Users/pacu/Repos/Zcash/wallet/zallet)
error[E0433]: failed to resolve: use of undeclared crate or module `xdg`
   --> zallet/src/commands/migrate_zcash_conf.rs:200:9
    |
200 |         xdg::BaseDirectories::new()
    |         ^^^ use of undeclared crate or module `xdg`

```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -23,6 +23,7 @@ tokio = "1"
 
 # Filesystem
 home = "0.5"
+xdg = "2.5"
 
 # Localization
 i18n-embed = { version = "0.15", features = ["fluent-system"] }

--- a/zallet/Cargo.toml
+++ b/zallet/Cargo.toml
@@ -39,6 +39,7 @@ toml.workspace = true
 tonic.workspace = true
 tower = { workspace = true, features = ["timeout"] }
 transparent.workspace = true
+xdg.workspace = true
 zcash_client_backend = { workspace = true, features = [
     "lightwalletd-tonic-tls-webpki-roots",
     "orchard",

